README.md

```diff
@@ -1,7 +1,7 @@
-# embedded-sgp
+# embedded-sg 
 Embedded SGP Drivers
 
-## Clone this repository
+## Clone this repository 
 ```
 git clone --recursive https://github.com/Sensirion/embedded-sgp.git
 ```
```

Comment:
Official sensor name is capital. Hence I would write: SGP30 driver related and SGPC3 driver related. In the fourth item you already wrote it with capital letters.